examples/simple-udp/main.go

```diff
@@ -36,17 +36,15 @@ func main() {
 	go doSignal()
 
 	port := flag.String("port", "7654", "The port to listen to udp traffic on")
+	passthrough := flag.Bool("passthrough", false, "Get listening port from the SDK, rather than use the 'port' value")
 	flag.Parse()
 	if ep := os.Getenv("PORT"); ep != "" {
 		port = &ep
 	}
-
-	log.Printf("Starting UDP server, listening on port %s", *port)
-	conn, err := net.ListenPacket("udp", ":"+*port)
-	if err != nil {
-		log.Fatalf("Could not start udp server: %v", err)
+	if epass := os.Getenv("PASSTHROUGH"); epass != "" {
```

Comment:
@ilkercelikyilmaz side thought - we could use flags like this to do your "shutdown after n minutes" load test - rather than you having to reload all your changes in on every change. WDYT?

Reply:
Agreed. That's what I had in mind as well.